storage/innobase/xtrabackup/test/suites/binlog/ib_slave_info.sh

```diff
@@ -165,7 +171,7 @@ setup_slave GTID $slave2_id $master_id
 mysql -e "SET GLOBAL general_log=1; SET GLOBAL log_output='TABLE';"
 
 vlog "Full backup of the GTID with AUTO_POSITION slave server"
-xtrabackup --backup --lock-ddl=false --slave-info --target-dir=$topdir/backup
+xtrabackup --backup --lock-ddl=OFF --slave-info --target-dir=$topdir/backup
```

Comment:
Lets kep OFF here. (Everwhere else, we can use false)

Reply:
AB